admin/javascript/src/components/form-builder/index.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as schemaActions from 'state/schema/actions';
+import SilverStripeComponent from 'silverstripe-component';
+import FormComponent from 'components/form/index';
+import TextField from 'components/text-field/index';
+import HiddenField from 'components/hidden-field/index';
+import GridField from 'components/grid-field/index';
+import fetch from 'isomorphic-fetch';
```

Comment:
I've had another look, and `isomorphic-fetch` doesn't actually add any code on top of `whatwg-fetch`, it simply makes it consumable by browserify in CommonJS format. So we'll stick with that for now.